components/StoryblokProvider.tsx

```diff
@@ -57,15 +57,42 @@ const components = {
   sbWysiwyg: SbWysiwyg,
 };
 
-storyblokInit({
-  accessToken: process.env.STORYBLOK_ACCESS_TOKEN,
-  use: [apiPlugin],
-  apiOptions: {
-    region: 'us',
-  },
-  components,
-});
+interface ProviderProps {
+  children: React.ReactNode;
+  isEditor?: boolean;
+};
+
+export default function StoryblokProvider({ children, isEditor = false }: ProviderProps) {
+
+  let accessToken = ''; // No access token because this is in client side code.
+  if (isEditor) {
+    if (typeof window !== 'undefined') {
+      const urlParams = new URLSearchParams(window.location.search);
+      accessToken = urlParams.get('token') || accessToken;
+    }
+  }
```

Comment:
If there is an accessToken available in the URL params and editor is flagged as true, use it.